new-lamassu-admin/src/pages/Transactions/Transactions.js

```diff
@@ -117,7 +117,7 @@ const Transactions = () => {
     },
     {
       header: 'Cash',
-      width: 104,
+      width: 110,
```

Comment:
How many more chars we can fit in this amount of pixels?

Reply:
If the amount reaches 4 digits it goes to a new line. Should I increase the width even more?